charts/tidb-cluster/values.yaml

```diff
@@ -261,6 +261,7 @@ tikv:
   # Specify the priorityClassName for TiKV Pod.
   # refer to https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#how-to-use-priority-and-preemption
   priorityClassName: ""
+  maxFailoverCount: 3
```

Comment:
Suggest to add comments here to make it clear what's the detailed meaning and maybe how to configure the value, e.g. for the cluster with 100 TiKV, what's the suggested config.

Reply:
Done.